app/assets/stylesheets/spotlight/_header.scss

```diff
@@ -31,23 +28,38 @@ $masthead-image-blur: 1px;
   }
 }
 
-#exhibit-masthead,
-#exhibit-navbar {
-  z-index: 1;
+#header-navbar {
+  margin-bottom: 0;
 }
 
-#exhibit-navbar {
-  z-index: 100;
+.masthead {
+  background-color: $masthead-bg;
+  border-bottom: 1px solid $navbar-default-border;
+  margin-bottom: $padding-large-vertical * 2.5;
+  overflow: hidden;
+  padding: 0;
+  position: relative;
+  // allow dropdowns in the top nav to stack above the masthead
+  z-index: $zindex-navbar - 1;
 
-  &.page-masthead {
-    @include transparent-masthead-navigation-menu();
-    background-color: $navbar-transparent-page-bg;
-    border-bottom: 1px solid $navbar-transparent-border;
-    margin-bottom: 0;
-    margin-top: 0;
 
-    .navbar-nav {
-      float: right;
+  .site-title {
+    position: relative;
+    white-space: nowrap;
+    padding-bottom: $padding-base-vertical;
+  }
+
+  small {
+    display: block;
+    margin-top: $padding-base-vertical;
+    padding-top: $padding-base-vertical;
+    padding-bottom: $padding-base-vertical;
+    @extend .hidden-xs;
```

Comment:
Expected item on line 57 to appear before line 53. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets